static/app/views/settings/organization/navigationConfiguration.tsx

```diff
@@ -1,158 +1,174 @@
-        show: ({features}) => features!.has('legacy-rate-limits'),
-        description: t('Configure rate limits for all projects in the organization'),
-        id: 'rate-limits',
-      },
-      {
-        path: `${organizationSettingsPathPrefix}/relay/`,
-        title: t('Relay'),
-        description: t('Manage relays connected to the organization'),
-        id: 'relay',
-      },
-      {
-        path: `${organizationSettingsPathPrefix}/repos/`,
-        title: t('Repositories'),
-        description: t('Manage repositories connected to the organization'),
-        id: 'repos',
-      },
-      {
-        path: `${organizationSettingsPathPrefix}/integrations/`,
-        title: t('Integrations'),
-        description: t(
-          'Manage organization-level integrations, including: Slack, Github, Bitbucket, Jira, and Azure DevOps'
-        ),
-        id: 'integrations',
-        recordAnalytics: true,
-      },
-      {
-        path: `${organizationSettingsPathPrefix}/early-features/`,
-        title: t('Early Features'),
-        description: t('Manage early access features'),
-        badge: () => <FeatureBadge type="new" />,
-        show: ({isSelfHosted}) => isSelfHosted || false,
-        id: 'early-features',
-        recordAnalytics: true,
-      },
-      {
-        path: `${organizationSettingsPathPrefix}/dynamic-sampling/`,
-        title: t('Dynamic Sampling'),
-        description: t('Manage your sampling rate'),
-        badge: () => <FeatureBadge type="alpha" />,
-        show: ({organization}) =>
-          !!organization && hasDynamicSamplingCustomFeature(organization),
-      },
-      {
-        path: `${organizationSettingsPathPrefix}/feature-flags/`,
-        title: t('Feature Flags'),
-        description: t('Set up your provider webhooks'),
-        badge: () => <FeatureBadge type="beta" />,
-        show: ({organization}) =>
-          !!organization && organization.features.includes('feature-flag-ui'),
-      },
-      {
-        path: `${organizationSettingsPathPrefix}/stats/`,
-        title: t('Stats & Usage'),
-        description: t('View organization stats and usage'),
-        id: 'stats',
-        show: ({organization}) =>
-          organization?.features.includes('navigation-sidebar-v2') ?? false,
-      },
-    ],
-  },
-  {
-    name: t('Developer Settings'),
-    items: [
-      {
-        path: `${organizationSettingsPathPrefix}/auth-tokens/`,
-        title: t('Auth Tokens'),
-        description: t('Manage organization auth tokens'),
-        id: 'auth-tokens',
-      },
-      {
-        path: `${organizationSettingsPathPrefix}/developer-settings/`,
-        title: t('Custom Integrations'),
-        description: t('Manage custom integrations'),
-        id: 'developer-settings',
-      },
-    ],
-  },
-];
+type ConfigParams = {
+  organization?: Organization;
+};
 
-export default organizationNavigation;
+export function getOrganizationNavigationConfiguration({
```

Comment:
Had to convert this to a function that accepts `organization` and returns the merged config if the feature flag is on. No other changes despite the giant diff